qtpy/QtGui.py

```diff
@@ -8,14 +8,27 @@
 
 """Provides QtGui classes and functions."""
 
-from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6
+from . import PYQT6, PYQT5, PYSIDE2, PYSIDE6, QtModuleNotInstalledError
+
+
+MISSING_OPENGL_NAMES = {}
```

Comment:
Is `MISSING_OPENGL_NAMES` still the right name for this variable? This mechanism should be able to handle deferring the error from any imports, not just OpenGL stuff.

I did briefly look for other modules that I might want to strip for size and didn't see any. I think this is only an issue if `qtpy` crosses Qt module boundaries, and everything else I saw was already properly contained.